Gemfile

```diff
@@ -17,7 +17,7 @@ gem 'audited', '>= 4.9.0', '< 5'
 gem 'will_paginate', '>= 3.1.7', '< 4'
 gem 'ancestry', '>= 3.0.7', '< 4', '!= 3.2.0'
 gem 'scoped_search', '>= 4.1.8', '< 5'
-gem 'ldap_fluff', '>= 0.4.7', '< 1.0'
+gem 'ldap_fluff', '>= 0.5.0', '< 1.0'
```

Comment:
how about bumping net-ldap version as well while were at it?

Reply:
I wanted to, but it's not strictly relevant and lzap already pushed the packaging PR. I can do another one just for the `net-ldap` :thinking:

Reply:
Added just as follow-up: https://github.com/theforeman/foreman-packaging/pull/6150

Reply:
BTW ideally I'd drop `net-ldap` and hide it behind `ldap_fluff` completely - keep it only as `ldap_fluff` dependency

Reply:
generally that would be OK but currently ldap_fluff requires >= 0.11 and we bumped it to >= 0.16 a few years ago due to a cve fix. if ldap_fluff is updated to the latest as well i'd be fine with it.

Reply:
As long as we deploy 0.16 or newer (and don't ship a GEMFILE lock), I don't think the lower bound should matter here, as long as it's compatible?

Reply:
Yeah, I guess it's not as big of a deal... it's not like we will downgrade the package to older then 0.11 in our packaging

Reply:
I can do so in the followup PR tho :)